test/unit/operations/aggregate.test.js

```diff
@@ -69,4 +72,71 @@ describe('AggregateOperation', function () {
       });
     });
   });
+
+  context('bypass validation', () => {
+    const test = {};
+    beforeEach(() =>
+      createServer().then(server => {
+        test.server = server;
+      })
+    );
+    afterEach(() => cleanup());
+
+    // general test for aggregate function
+    function testAggregate(config, done) {
+      const client = new MongoClient(`mongodb://${test.server.uri()}/test`);
+      let close = e => {
+        close = () => {};
+        client.close(() => done(e));
+      };
+
+      test.server.setMessageHandler(request => {
+        const doc = request.document;
+        if (doc.aggregate) {
+          try {
+            expect(doc.bypassDocumentValidation).equal(config.expected);
+            request.reply({
+              ok: 1,
+              cursor: {
+                firstBatch: [{}],
+                id: 0,
+                ns: 'test.test'
+              }
+            });
+          } catch (e) {
+            close(e);
+          }
+        }
+
+        if (isHello(doc)) {
+          request.reply(Object.assign({}, HELLO));
+        } else if (doc.endSessions) {
+          request.reply({ ok: 1 });
+        }
+      });
+
+      client.connect(function (err, client) {
+        expect(err).to.not.exist;
+        const db = client.db('test');
+        const collection = db.collection('test_c');
+
+        const options = { bypassDocumentValidation: config.actual };
+
+        const pipeline = [
+          {
+            $project: {}
+          }
+        ];
+        collection.aggregate(pipeline, options).next(() => close());
```

Comment:
I'm just going to respond here for the next four comments, since they're all the same.  I agree that it makes sense to move the tests to `collection.test.js`.  I'll make that change!